central/metrics/central.go

```diff
@@ -182,6 +183,11 @@ var (
 		Name:      "pipeline_panics",
 		Help:      "A counter that tracks the number of panics that have occurred in the processing pipelines",
 	}, []string{"resource"})
+	sensorConnectedCounter = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: metrics.PrometheusNamespace,
+		Subsystem: metrics.CentralSubsystem.String(),
+		Name:      "sensor_connected",
+	}, []string{"ClusterID", "reconnect"})
```

Comment:
It can be useful only temporarily, but let's maybe record the state of the feature flag here as an additional label?

```suggestion
	sensorConnectedCounter = prometheus.NewCounterVec(prometheus.CounterOpts{
		Namespace: metrics.PrometheusNamespace,
		Subsystem: metrics.CentralSubsystem.String(),
		Name:      "sensor_connected",
	}, []string{"ClusterID", "reconnect", "restartPreventionEnabled"})
```

Reply:
Added to the metric